src/FSharp.AWS.DynamoDB/TableContext.fs

```diff
@@ -343,11 +332,10 @@ type TableContext<'TRecord> internal
         let writeRequests = items |> Seq.map mkWriteRequest |> rlist
         let pbr = BatchWriteItemRequest()
         pbr.RequestItems.[tableName] <- writeRequests
-        let returnConsumedCapacity, maybeReport = metricsOptions collector
         pbr.ReturnConsumedCapacity <- returnConsumedCapacity
```

Comment:
I guess having an explicit line aids scanning the code, but it may be worth collapsing these into the construction of the Request, i.e. 
`let pbr = BatchWriteItemRequest(ReturnConsumedCapacity = returnConsumedCapacity)`